lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -205,19 +205,19 @@ def __get_k8s_job_spec_template(self, ajs):
         k8s_spec_template = {
             "metadata": {
                 "labels": {
-                    "app.kubernetes.io/name": ajs.job_wrapper.tool.old_id,
-                    "app.kubernetes.io/instance": self.__produce_k8s_job_prefix(),
-                    "app.kubernetes.io/version": ajs.job_wrapper.tool.version,
                     "app.kubernetes.io/component": "tool",
                     "app.kubernetes.io/part-of": "galaxy",
                     "app.kubernetes.io/managed-by": "galaxy",
-                    "app.galaxyproject.org/job_id": ajs.job_wrapper.get_id_tag(),
-                    "app.galaxyproject.org/instance": self._galaxy_instance_id or "",
-                    "app.galaxyproject.org/handler": self.app.config.server_name,
-                    "app.galaxyproject.org/destination": ajs.job_wrapper.job_destination.id,
```

Comment:
job_id, handler, and destination are important as labels so that selectors can be written to reference them

Reply:
Were there any particular use cases you had in mind?

I've implemented the regex solution for now, just to get past the immediate roadblock, but it's probably worth rationalizing whether each non-standard item should be a label or an annotation.

Reply:
The rational is that if you have 20K jobs running and you need to debug one of them, to provide a means of filtering on useful properties.

Reply:
This info is also queryable from the database. Creating a secondary queryable database in Kubernetes does open up some new possibilities,  but it does have a small cost and I think that motivating it with concrete usecases would be helpful in determining what to include.  E.g. would it be useful to add the user id in there? Do we need to filter by workflow etc.

Reply:
This information is important for cluster statistics and compute quota enforcement. De-duplicating this information is a bit of an over optimization.